README.md

```diff
@@ -115,6 +107,14 @@ From this point on, we'll assume that your web server is available at this local
 
 http://localhost:8000/
 
+Following are the routes to access our apis.
+
+Author API : http://localhost:8000/authorapi
+Questions API : http://localhost:8000/questionsapi
+Items API : http://localhost:8000/itemsapi
+Reports API : http://localhost:8000/reportsapi
+Question Editor API : http://localhost:8000/questioneditorapi
+
```

Comment:
@bhavya-shukla-lrn we should perhaps update the following line of documentation, since you have now added many more examples of functionality. We could replace line 118 with the following: 

"Open these pages with your web browser. These are all basic examples of  Learnosity's integration. You can interact with these demo pages to try out the various APIs. The Items API example is a basic example of an assessment loaded into a web page with Learnosity's assessment player. You can interact with this demo assessment to try out the various Question types."